File: nara/python/01/00.py
def reverse_s(forward_string: str) -> str:
    r_s = ""
    for i in s:
        r_s = i + r_s
    return r_s


if __name__ == "__main__":
    s = "stressed"
    rev_s = reverse_s(s)
    print(rev_s)

File: nara/cc/01/00.cpp
#include <iostream>
using namespace std;

string reverse_s(string s) {
    int n = s.length();
    string r_s = "";
    for (int i = 0; i < n; ++i) {
        r_s = s[i] + r_s;
    }
    return r_s;
}

int main() {
    string s = "stressed";
    string rev_s = reverse_s(s);
    cout << rev_s << endl;
}